recipe/meta.yaml

```diff
@@ -23,8 +33,8 @@ requirements:
     - pip
     - wheel
     - setuptools
-    # See our custom cbc.yaml for modified pinnnigs
-    - numpy
+    # See above for pinning details
+    - numpy {{ numpy }}
     - scikit-learn >=0.20.0
```

Comment:
Serhii asked if these `host` constraints were truly necessary. Unfortunately, they are. This line in the setup.py:

https://github.com/scikit-learn-contrib/category_encoders/blob/master/setup.py#L4

causes the file `category_encoders/__init__.py` to be imported:

https://github.com/scikit-learn-contrib/category_encoders/blob/master/category_encoders/__init__.py

And this in turn brings in virtually the entire dependency set.